server/routes/prisons/exclusionDates/exclusionDatesController.ts

```diff
@@ -1,12 +1,76 @@
 import { RequestHandler } from 'express'
 import { PrisonService } from '../../../services'
 
+const MONTHS = [
+  'January',
+  'February',
+  'March',
+  'April',
+  'May',
+  'June',
+  'July',
+  'August',
+  'September',
+  'October',
+  'November',
+  'December',
+] as const
+
+const transformDate = (date: string): string => {
+  const splitedDate = date.split('-').map(d => Number(d))
```

Comment:
You could use https://date-fns.org/docs/Getting-Started for this. There's a few standard date formatting functions we've created in one of the utils files I believe.